File: python/7 kyu/440.py
# https://www.codewars.com/kata/56269eb78ad2e4ced1000013/train/python

# Passed

def find_next_square(sq):
    square_root = sq ** 0.5
    result = -1
    if square_root.is_integer():
        result = int((square_root + 1) ** 2)
    
    return result

output = find_next_square(625)
print(output)

File: nodejs/7 kyu/440.js
// https://www.codewars.com/kata/57d1f36705c186d018000813/train/javascript

// Passed

function gordon(a) {
  let vowels = new Set("AEIOU");
  let wordToGordonWord = (word) => {
    let gordonWord = "";
    for (const char of word.toUpperCase()) {
      gordonWord += char === "A" ? "@" : vowels.has(char) ? "*" : char;
    }

    gordonWord += "!!!!";
    return gordonWord;
  };
  let words = a.split(" ").map(wordToGordonWord);
  let result = words.join(" ");
  return result;
}

const output = gordon("i am a chef");
console.log(output);